fe/fe-core/src/main/java/org/apache/doris/tablefunction/ExternalFileTableValuedFunction.java

```diff
@@ -324,4 +323,13 @@ private PFetchTableSchemaRequest getFetchTableStructureRequest() throws Analysis
         return InternalService.PFetchTableSchemaRequest.newBuilder()
                 .setFileScanRange(ByteString.copyFrom(new TSerializer().serialize(fileScanRange))).build();
     }
+
+    protected Backend getBackend() {
```

Comment:
It should return a `List<Backend>`, because for a tvf, it can be send to multi backends to execute.
And `local()` tvf is a special case that the size of list is 1